src/app/hooks/useViewTracker/index.test.jsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { useInView } from 'react-intersection-observer';
+
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import useViewTracker from '.';
+
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import pageData from './pageData.json';
+
+delete window.location;
+window.location = { href: 'http://bbc.com/pidgin/tori-51745682' };
+process.env.SIMORGH_ATI_BASE_URL = 'https://logws1363.ati-host.net?';
+
+jest.mock('react-intersection-observer');
+
+beforeEach(() => {
+  jest.clearAllMocks();
+  jest.useFakeTimers();
+});
+
+afterEach(() => {
+  jest.runOnlyPendingTimers();
+  jest.useRealTimers();
+});
+
+const elementRefFn = jest.fn();
```

Comment:
```suggestion
const observedElementRef = jest.fn();
```
I found this variable name a little confusing (unless I'm completely missing it's purpose) is it a "`ref` referring to the element being observed"? So in other words, the element being tracked for views?

Reply:
It's a ref that should be set on the DOM element. Maybe just `elementRef` would be less confusing?

Reply:
`elementRef` sounds good